src/simulator/single_publisher_strategy.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A trivial modeling strategy for use with single publisher models."""
+
+from typing import Dict
+from typing import Type
+
+from wfa_planning_evaluation_framework.models.reach_curve import ReachCurve
+from wfa_planning_evaluation_framework.models.reach_surface import ReachSurface
+from wfa_planning_evaluation_framework.simulator.halo_simulator import HaloSimulator
+from wfa_planning_evaluation_framework.simulator.modeling_strategy import (
+    ModelingStrategy,
+)
+from wfa_planning_evaluation_framework.simulator.privacy_tracker import PrivacyBudget
+from wfa_planning_evaluation_framework.simulator.system_parameters import (
+    SystemParameters,
+)
+from wfa_planning_evaluation_framework.simulator.privacy_tracker import (
+    PrivacyBudget,
```

Comment:
Is there any particular reason to import `PrivacyBudget` twice?

Reply:
No.  Thanks for catching that.